### PR TITLE
Added foreman-discovery-image git repo

### PR DESCRIPTION
**Pull request by lzap**
Thanks @domcleal

### CODE REVIEW

File: .openshift/cron/hourly/git_repos [hunk 1]
```diff
@@ -61,6 +61,7 @@ update_repo smart_proxy_remote_execution_ssh https://github.com/theforeman/smart
 update_repo smart_proxy_salt https://github.com/theforeman/smart_proxy_salt
 update_repo stacker https://github.com/ohadlevy/stacker.git
 update_repo foreman_deployments https://github.com/theforeman/foreman_deployments
+update_repo foreman-discovery-image https://github.com/theforeman/foreman-discovery-image
```
Comment (domcleal) [inline .openshift/cron/hourly/git_repos]: Would you mind re-alphabetising this?  I didn't realise it stopped.

Comment (lzap) [inline .openshift/cron/hourly/git_repos]: Sure, added an extra commit for verbosity.

**Comment by domcleal** [comment]
Thanks @lzap, merging and pushing to OpenShift.  It should get created within an hour or so, you can add it to Redmine's settings.

**Comment by lzap** [comment]
I don't have much permissions on RedMine, I only see Versions and Wiki in the Settings page. Where is that?

**Comment by domcleal** [comment]
Ah, @GregSutcliffe is the only manager of the Discovery project.  I did it instead.  If he'd like you to manager, he can.

**Comment by lzap** [comment]
Thanks.

**Pull request merged**